toolkit/imageconfigs/packagelists/azurevm-packages.json

```diff
@@ -11,6 +11,7 @@
         "rsyslog",
         "sgx-backwards-compatability",
         "WALinuxAgent",
-        "wireless-regdb"
+        "wireless-regdb",
+        "kexec-tools"
```

Comment:
nit - alphabetical

Reply:
I agree with Chris, I would be nice to have them in alphabetical order (easier to maintain and check)

Reply:
Updated. Thanks!